app/api-v1/controllers/Recipe/__tests__/transaction_fixtures.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  recipeId: 'RECIPE00-0000-1000-8000-000000000000',
+  recipeExample: {
+    id: 'RECIPE00-9000-1000-8000-000000000000',
+    price: '99.99',
+    material: 'iron',
+    supplier: 'supplier-address',
+  },
+  transactionsExample: [
+    {
+      id: 'TRASACTI-0000-1000-8000-000000000000',
+      recipe_id: 'RECIPE00-0000-1000-8000-000000000000',
+      token_id: 'token-id',
+      status: 'InBlock',
```

Comment:
not a valid status according to the table enum?

Reply:
table should have bee updated 🤔

Reply:
Mmm.. you are right, updating along with integration test